react/LocationSearch.tsx

```diff
@@ -27,11 +27,18 @@ const getAddresses = (searchTerm: string) => {
 
 interface LocationSearchProps {
   onSelectAddress?: (selectedAddress: string) => void
+  renderEngineLogo?: (
+    props: React.ImgHTMLAttributes<HTMLImageElement> & { className?: string }
+  ) => HTMLImageElement
```

Comment:
hmmm, I don't know how I feel about this parameter and return value.. I think it's best to let the user of this component decide what to render here, no? in case we want to render a svg instead of a img for example

Reply:
Agreed, good point. I'm just unsure on how to do that. Does `renderEngineLogo?: (props: React.HTMLAttributes<HTMLElement>) => HTMLElement` makes sense? 🤔